### DISCUSSION TITLE
Suddenly several of my apps has faced many crashes on 22/03/2021 & 23/03/2021. Does anyone else face this ?

### DISCUSSION THREAD

**Discussion by Gokulv617** [airsdk/Adobe-Runtime-Support#750] [General]
Suddenly several of my apps in playstore has many crashes in these 2 days **22/03/2021 & 23/03/2021**. I didn't had any changes for the last 30 days to any of the apps. Also I have investigated the apps but could find any crashes. Some of the users have reported me that _"while loading the app it kicks me out suddenly, but before it was fine"_.

Does anyone else face this situation ?

![501-2](https://user-images.githubusercontent.com/43432782/112419642-c14f5000-8d51-11eb-94c7-17c3abb5ce92.png)

![rev](https://user-images.githubusercontent.com/43432782/112419696-db892e00-8d51-11eb-84ea-275891b89332.png)

![sur](https://user-images.githubusercontent.com/43432782/112419893-37ec4d80-8d52-11eb-8d1b-a60868327f37.png)

![DF](https://user-images.githubusercontent.com/43432782/112420207-d7114500-8d52-11eb-98f5-5d44daada983.png)

![LS](https://user-images.githubusercontent.com/43432782/112420280-f9a35e00-8d52-11eb-9105-a1e49137949e.png)

![100doors](https://user-images.githubusercontent.com/43432782/112420455-3e2ef980-8d53-11eb-84be-718d9754753b.png)

**Comment by ajwfrost** [comment]
Wow that's a serious change...
Are you able to confirm what version of the AIR SDK you used, and provide any of the crash reports so that we can check the call stacks?
thanks

**Reply by Gokulv617** [reply]
I use AIR 33.1.1.345 for all the mentioned projects. Happened mostly in ANDROID 11, ANDROID 10, ANDROID 9

**Crashes in last 7days**
![signal 11](https://user-images.githubusercontent.com/43432782/112427233-3d9c6000-8d5f-11eb-8d1b-0ec1aef66e9f.png)

### STACK TRACE: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR)
pid: 0, tid: 0 >>> air.com.app.myapp<<<

backtrace:
`  #00  pc 0000000002919eba  /data/app/com.google.android.trichromelibrary_438909030-fSz60XsHTQgcIvrm4zlfbQ==/base.apk!libmonochrome.so (offset 0x645000)`

### STACK TRACE: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR)
pid: 0, tid: 0 >>> air.com.app.myapp<<<

backtrace:
`  #00  pc 00000000034521d0  /data/app/com.android.chrome-zlKcqC2ZjDqioTByiuQC1g==/base.apk (offset 0xa0f000)`

**Comment by Gokulv617** [comment]
I'm seriously monitoring and maintaining this app's ANR's and Crashes. 

### Actual crash status as of now of this app
![501-2C](https://user-images.githubusercontent.com/43432782/112428311-fa42f100-8d60-11eb-8fcd-801691f58646.png)

### Actual ANR status as of now of this app
![501-2A](https://user-images.githubusercontent.com/43432782/112428511-4726c780-8d61-11eb-93dc-3eb009739070.png)

Suddenly a huge spike in crashes which reached above 2K in last 3 days. These crashes from somewhere is going to affect the crash rate percent. Please help me soon

**Comment by ajwfrost** [comment]
Hi

The crash report showing as within libmonochrome - this sounds like the Android WebView problem:
https://stackoverflow.com/questions/66361472/app-crashed-with-base-apklibmonochrome-so
https://www.bbc.co.uk/news/technology-56496783

Not sure if there's a way you can adjust the app for the short term to avoid the use of this that is causing the problem, but hopefully the fix from Google will roll out and you'll see the numbers drop again..

thanks

**Comment by Gokulv617** [comment]
### Thank you so much. This helps me a lot
